### PR TITLE
1.2.11

### PR DESCRIPTION
**Pull request by humanagent**
## Release Notes

**TLDR**

- CDP integration
- Framesv2 integration
- Updated node-sdk bindings

- **New Features**
	- Introduced USDC management capabilities: check balances, fund wallets, and transfer USDC.
	- Added a new `WalletService` for managing agent wallets, including gasless payments and transfers.
	- New methods for wallet management: `requestFunds`, `checkBalance`, `withdrawFunds`, and `swap`.
	- Documentation enhancements for USDC and wallet management, including a new "Agent Wallet" section.
	- Added a new script for development in the `templates/toss` directory.
	- Enhanced `RunConfig` to specify a Redis client for wallet service interactions.

- **Bug Fixes**
	- Improved error handling and logging for wallet operations and toss management.

- **Documentation**
	- Expanded sections on file structure, wallet service functionalities, and added clear instructions for new environment variables.
	- Updated documentation to clarify the inclusion of the OpenAI package and API key requirements.
	- Enhanced clarity in the `resolver` documentation regarding code imports from MessageKit.

### CODE REVIEW

**Comment by humanagent** [comment]
@coderabbitai pause

File: packages/message-kit/src/helpers/cdp.ts [hunk 1]
```diff
@@ -1,124 +1,330 @@
...
+  private hexToBytes(hex: string): Uint8Array {
+    if (hex.startsWith("0x")) {
+      hex = hex.slice(2);
+    }
+    const bytes = new Uint8Array(hex.length / 2);
+    for (let i = 0; i < hex.length; i += 2) {
+      bytes[i / 2] = parseInt(hex.substr(i, 2), 16);
+    }
+    return bytes;
+  }
```
Comment (rygine) [inline packages/message-kit/src/helpers/cdp.ts:77]: would `toBytes` from `viem` work for this instead?

**Review by rygine — APPROVED** [review]
